kolibri/plugins/learn/options.py

```diff
@@ -0,0 +1,9 @@
+option_spec = {
+    "LEARN": {
```

Comment:
If you're editing the default, can also make this Title Case, which seems to be our unwritten convention for options section names.